src/clients/SpokePoolClient.ts

```diff
@@ -648,14 +648,12 @@ export class SpokePoolClient extends BaseAbstractClient {
       for (const event of slowFillRequests) {
         const slowFillRequest = {
           ...spreadEventWithBlockNumber(event),
+          messageHash: getMessageHash(event.args["message"]),
```

Comment:
Can we add a simple unit test for this addition?

Reply:
https://github.com/across-protocol/sdk/pull/844/commits/477d52147e70395d4ecdf9d098d431d31501a013